bagitobjecttransfer/bagitobjecttransfer/settings/base.py

```diff
@@ -243,6 +243,12 @@
             ),
             'output_filename': 'recordtransfer/js/profile/min.js',
         },
+        'recordtransfer_admin_js': {
+            'source_filenames': (
+                'recordtransfer/js/admin/*.js',
+            ),
+            'output_filename': 'recordtransfer/js/admin/min.js',
+        },
```

Comment:
There may need to be a `caais_admin_js` section here for the CAAIS JavaScript

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/221/commits/97e8fea12190524d00c24359325f8162dc7ef037